File: C++/Contoh Statement/Contoh If, Else If, dan Else/If, Else If, dan Else.cpp
#include <iostream>

using namespace std;

int main()
{
	int pil;
	cout << "Menu :\n" << endl;
	cout << "Masukan Angka : 1 Contoh If" << endl;
	cout << "Masukan Angka : 2 Contoh Else If" << endl;
	cout << "Masukan Angka Selain 1 dan 2 Untuk Contoh Else\n" << endl;
	cout << "Silahkan Masukan Angka : ";
	cin >> pil;

	if (pil == 1) {
		cout << "\nini Adalah Contoh If" << endl;
	}
	else if (pil == 2) {
		cout << "\nini Adalah Contoh Else If" << endl;
	}
	else {
		cout << "\nini Adalah Contoh Else" << endl;
	}

	return 0;
}

File: Kotlin/Contoh Statement/Contoh If, Else If, dan Else/If, Else If, dan Else.kt
import java.util.Scanner

fun main(args: Array<String>) {
    var input = Scanner(System.`in`)

    println("Menu \n")
    println("Masukan Angka : 1 Contoh If")
    println("Masukan Angka : 2 Contoh Else If")
    println("Masukan Angka Selain 1 dan 2 Untuk Contoh Else\n")
    print("Silahkan Masukan Angka : ")
    var pil = input.nextInt()

    if (pil == 1){
        println("\nini Adalah Contoh If")
    }
    else if (pil == 2){
        println("\nini Adalah Contoh Else If")
    }
    else {
        println("\nini Adalah Contoh Else")
    }
}
